fastlane_core/lib/fastlane_core/itunes_transporter.rb

```diff
@@ -200,7 +200,8 @@ def build_download_command(username, password, apple_id, destination = "/tmp")
     end
 
     def build_upload_command(username, password, source = "/tmp")
-      [
+      short_team_id = (ENV['FASTLANE_SHORT_TEAM_ID'] || '').strip
```

Comment:
Right. Forgot about the `upload` command - never used it.

I got this pattern from the spaceship lib [here](https://github.com/fastlane/fastlane/blob/435f4847731be666e4f011f5093ab1a5331940c1/spaceship/lib/spaceship/tunes/tunes_client.rb#L79)

Does it make sense to set this value in the `initialize` method from the environment or did you have something different in mind? I'd like to have this value just come from the `Appfile`